texts/epn_paper.md

```diff
@@ -0,0 +1,71 @@
+
+Your routine changed drastically over the course of the last couple of months due to the lockdowns. You now mostly stay at home, hopefully safe and healthy.
+Your classes are now online, and probably you had to quickly figure out how run them at all.
+If you do experiments, your lab is either closed or operates at a much lower capacity.
+Your travel plans are cancelled. Still, despite that, you have likely attended a scientific presentation or even gave one yourself.
+The only difference is that it happened online.
+
+<!-- #### Online talks were gaining momentum already before, and for a good reason. -->
+
+Virtual conferences were gaining momentum even before the coronavirus crisis due to multiple advantages over traditional in-person gatherings.
+With the increasing concerns about greenhouse emissions, many prefer to limit their long-distance travel.
+To some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether.
+Finally, most of the expensive and time-consuming aspects of conference organisation, such as finding a venue, lodging and catering for the participants are gone entirely.
+With all these obstacles out of the way, virtual conferences can be organized much faster and respond to the latest developments.
+They are also more inclusive and reach a broader audience especially if the talks are recorded.
+
+![](planning_checklist.png)
+
+**Figure caption:** A conference organizer checklist.
+
+<!-- #### We decided to develop online conferences and established VSF. -->
+
+After finding each other as like-minded individuals with an urge to improve the present situation, we started gathering in the late summer of 2019 to explore our options. 
+It quickly became clear that we had the capacity to establish a venue for virtual conferencing. 
+So, we committed ourselves to develop this idea into a real platform, which we now call the Virtual Science Forum.
+
+<!-- #### In thinking about online events we had to revisit many assumptions about conferences. -->
+
+As we started to organize ourselves, it became clear quickly that in order to achieve our goals, we needed to revisit our assumptions about online events.
+For example, the interactions between the speaker and the audience as well as among the audience themselves in an online event are quite different and much less spontaneous.
+The coffee break, such a natural part of in-person conferences, does not function effectively in an online event to spark discussions.
+Likewise, the length of an online event is constrained by very different factors, such as the spread of timezones and local commitments and distractions of the participants.
+
+<!-- #### We ran a couple of events, and saw that they work well. -->
+
+We created toolkits and developed strategies and guidelines to navigate these challenges.
+And we got to work to organize online events, starting with a beta test workshop in December 2019.
```

Comment:
To navigate these challenges we built toolkits, developed strategies, and created guidelines.
We got to work on organizing our first online test workshop in December 2019, putting our guidelines and strategies to good use.

Reply:
Fixed in the latest version